tests/basic/storage_pool_test.go

```diff
@@ -6035,3 +6035,256 @@ outer:
+			dash.VerifyFatal(err, nil, "Pool expansion init successful?")
+
+			resizeErr := waitForPoolToBeResized(expectedSize, poolUUID, isjournal)
+			dash.VerifyFatal(resizeErr, nil, fmt.Sprintf("Verify pool [%s] on node [%s] expansion using auto", poolUUID, nodeDetail.Name))
+		})
+
+		// Validate if PoolIO Priority is not changed after pool Expansion
+		ioPriorityAfter, err := Inst().V.GetPoolLabelValue(poolUUID, "iopriority")
+		log.FailOnError(err, "Failed to get IO Priority for Pool with UUID [%v]", poolUUID)
+
+		log.InfoD(fmt.Sprintf("Priority Before [%s] and Priority after Pool Expansion [%s]", ioPriorityBefore, ioPriorityAfter))
+		dash.VerifyFatal(ioPriorityAfter == setIOPriority, true, "IO Priority mismatch after pool expansion")
+
+	})
+
+	JustAfterEach(func() {
+		defer EndTorpedoTest()
+		log.InfoD("Exit from Maintenance mode if Pool is still in Maintenance")
+		ExitNodesFromMaintenanceMode()
+		AfterEachTest(contexts, testrailID, runID)
+	})
+})
+
+var _ = Describe("{VerifyPoolDeleteInvalidPoolID}", func() {
+	var testrailID = 79487
+	// Testrail Description : Verify deletion of invalid pool ids
+	// Testrail Corresponds : https://portworx.testrail.net/index.php?/cases/view/55349
+
+	// Testrail Corresponds : https://portworx.testrail.net/index.php?/cases/view/55330
+	// Testrail Description : Delete pool when PX/Pool (2.6.0+) is not in maintenance mode and verify the error message
+
+	var runID int
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("VerifyPoolDeleteInvalidPoolID",
+			"Verify deletion of invalid pool ids",
+			nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	var contexts []*scheduler.Context
+	stepLog := "Verify deletion of invalid pool ids"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("deleteinvalidpoolid-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		defer appsValidateAndDestroy(contexts)
+
+		// Get the Pool UUID on which IO is running
+		poolUUID, err := GetPoolIDWithIOs()
+		log.FailOnError(err, "Failed to get pool using UUID [%v]", poolUUID)
+		log.InfoD("Pool UUID on which IO is running [%s]", poolUUID)
+
+		nodeDetail, err := GetNodeWithGivenPoolID(poolUUID)
+		log.FailOnError(err, "Failed to get Node Details from PoolUUID [%v]", poolUUID)
+
+		PoolDetail, err := GetPoolsDetailsOnNode(*nodeDetail)
+		log.FailOnError(err, "Fetching all pool details from the node [%v] failed ", nodeDetail.Name)
+		log.InfoD("Pool Details [%v] Picked for Delete", PoolDetail)
+
+		// Delete Pool without entering Maintenance Mode [ PTX-15157 ]
+		err = Inst().V.DeletePool(*nodeDetail, "0")
+		dash.VerifyFatal(err == nil, false, fmt.Sprintf("Pool Delete Status is SUCCESS. Expected Failure as pool not in maintenance mode : Node Detail [%v]", nodeDetail.Name))
```

Comment:
you can remove "Pool Delete Status is SUCCESS" coz if even if pool delete is failed, it will log as sucess

Reply:
fixed